<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkplaceWidget</class>
 <widget class="QWidget" name="WorkplaceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="userPage">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>User</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QTableView" name="userTableView"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="displayCartButton">
           <property name="text">
            <string>Display Cart</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="displayShopButton">
           <property name="text">
            <string>Display Shop</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addToCartButton">
           <property name="text">
            <string>Add to Cart</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeFromCartButton">
           <property name="text">
            <string>Remove from Cart</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="vendorPage">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>Vendor</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="2">
          <widget class="QPushButton" name="addGoodButton">
           <property name="text">
            <string>Add Good</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="goodPriceEdit"/>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="changeGoodButton">
           <property name="text">
            <string>Change Good</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="goodNameEdit"/>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="removeGoodButton">
           <property name="text">
            <string>Remove Good</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="goodNameLabel">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="goodPriceLabel">
           <property name="text">
            <string>Price</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="vendorTableView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="adminPage">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>Admin</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="horizontalSpacing">
          <number>10</number>
         </property>
         <item row="1" column="2">
          <widget class="QPushButton" name="changeUserButton">
           <property name="text">
            <string>Change User</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="removeUserButton">
           <property name="text">
            <string>Remove User</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="userNameLabel">
           <property name="text">
            <string>Username</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="userNameEdit"/>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="addUserButton">
           <property name="text">
            <string>Add User</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="userPasswordLabel">
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="userPasswordEdit"/>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="userAccessLvEdit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="userAccessLvLabel">
           <property name="text">
            <string>Access Level</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="adminTableView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
